array.with().md

```diff
@@ -0,0 +1,68 @@
+---
+ Title: 'array.prototype.with()'
+Description: 'This teaches JavaScript developers how to use the array method .with()'
+Subjects:
+  - 'Web development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  -'course/introduction-to-javascript'
+  -'paths/front-end-engineer-career-path'
+---
+
+## .with()
+
+The `.with()` **method** modifies the value of an element in an **array**. Since `.with()` is a **copying method**, it creates a new array. 
```

Comment:
```suggestion
The **`.with()`** method modifies the value of an element in an array. Since `.with()` is a copying method, it creates a new array. 
```

Reply:
Move this up to line 14